src/anemoi/datasets/create/__init__.py

```diff
@@ -487,6 +487,7 @@ def _run(self):
         assert chunks == self.dataset.get_zarr_chunks(), (chunks, self.dataset.get_zarr_chunks())
 
         def sanity_check_config(a, b):
+            return
```

Comment:
Why return here? Rest of code won't be used.
Should this be on line 498?

Reply:
The message below has confused a few users. So, I have temporarily disabled it.